vpdq/cpp/io/vpdqio.cpp

```diff
@@ -24,7 +25,7 @@ using namespace facebook::pdq::io;
 namespace facebook {
 namespace vpdq {
 namespace io {
-
+int PRECESION = 3;
```

Comment:
nit: spelling and variable description e.g.
`const int TIMESTAMP_OUTPUT_PRECISION = 3`